src/twisted/protocols/tls.py

```diff
@@ -682,6 +690,100 @@ def clientConnectionForTLS(self, protocol):
         return self._connectionForTLS(protocol)
 
 
+class AggregateSmallWrites:  # TODO make this private for now?
+    """
+    Aggregate small writes so they get written in large batches.
+
+    If this is used as part of a transport, the transport needs to call
+    ``flush()`` immediately when ``loseConnection()`` is called, otherwise any
+    buffered writes will never get written.
+    """
+
+    def __init__(self, write: Callable[[bytes], object], clock: IReactorTime):
+        self._write = write
+        self._clock = clock
+        self._buffer: list[bytes] = []
+        self._bufferLen = 0
+        self._scheduled: Optional[IDelayedCall] = None
+
+    def write(self, data: bytes) -> None:
+        """
+        Buffer the data, or write it immediately if we've accumulated enough to
+        make it worth it.
+
+        Accumulating too much data can result in higher memory usage.
+        """
+        self._buffer.append(data)
+        self._bufferLen += len(data)
+        if self._bufferLen > 64_000:
```

Comment:
Minor comment.

Maybe extract this value into a class member, and have it documented at the start of the class definition

```suggestion
        if self._bufferLen > self._MAX_CHUNK:
```

Reply:
+1